liquibase-standard/src/main/java/liquibase/statement/SqlStatement.java

```diff
@@ -23,5 +27,17 @@ public interface SqlStatement {
 
     boolean continueOnError();
 
+    default String getFormattedStatement(Database database) {
+//        try {
+
+        if (database != null) {
+            return SqlUtil.getSqlString(this, SqlGeneratorFactory.getInstance(), database)
+                    .replaceFirst(";$", "");
```

Comment:
Why are you doing this? `.replaceFirst(";$", "");`

This wasn't in the original code moved from `liquibase-standard/src/main/java/liquibase/changelog/ChangeSet.java`